internal/oc/release.go

```diff
@@ -129,19 +129,11 @@ func (r *release) getImageByName(log logrus.FieldLogger, imageName, releaseImage
 		return "", errors.New("neither releaseImage, nor releaseImageMirror are provided")
 	}
 
-	if releaseImageMirror != "" {
-		//TODO: Get mirror registry certificate from install-config
-		image, err = r.getImageFromRelease(log, imageName, releaseImageMirror, pullSecret, true)
-		if err != nil {
-			log.WithError(err).Errorf("failed to get %s image from mirror release image %s", imageName, releaseImageMirror)
-			return "", err
-		}
-	} else {
-		image, err = r.getImageFromRelease(log, imageName, releaseImage, pullSecret, false)
-		if err != nil {
-			log.WithError(err).Errorf("failed to get %s image from release image %s", imageName, releaseImage)
-			return "", err
-		}
+	//TODO: Get mirror registry certificate from install-config
+	image, err = r.getImageFromRelease(log, imageName, releaseImage, releaseImageMirror, pullSecret)
+	if err != nil {
+		log.WithError(err).Errorf("failed to get %s image from mirror release image %s or mirror %s", imageName, releaseImage, releaseImageMirror)
```

Comment:
Wouldn't be a misleading log traces in case of empty `releaseImageMirror` (previous implementation seems to provide a different logging message in that case)

Reply:
Updated log messages to be similar to previous implementation.